launch/client.py

```diff
@@ -219,13 +217,6 @@ def register_batch_csv_location_fn(
         """
         self.batch_csv_location_fn = batch_csv_location_fn
 
-    def register_endpoint_auth_decorator(self, endpoint_auth_decorator_fn):
```

Comment:
we can do this since v1 sets aws_role + results_s3_bucket based on if identity service says we're a scale user?

Reply:
Yup